azext_iot/digitaltwins/_help.py

```diff
@@ -113,6 +113,19 @@ def load_digitaltwins_help():
             az dt delete -n {instance_name} -y --no-wait
     """
 
+    helps["dt wait"] = """
+        type: command
+        short-summary: Wait until an operation on an Digital Twins instance is complete.
+
+        examples:
+        - name: Wait until an arbitrary instance is created.
+          text: >
+            az dt wait -n {instance_name} --created
+        - name: Wait until an existing instance is deleted.
+          text: >
+            az dt wait -n {instance_name} --deleted
```

Comment:
Can we add an example for `--custom`? I think the `--created` and `--deleted` are pretty self-explanatory, but an example for `--custom` would be super helpful

Reply:
done